Utilities/VSOBuildScripts/postBuildScript.cmd

```diff
@@ -1,22 +1,30 @@
 if "%1" == "local" goto testlocal
```

Comment:
You can also assume that if %BUILD_SOURCESDIRECTORY% is empty then its being run locally.